.changelog/16288.txt

```diff
@@ -0,0 +1,8 @@
+```release-note:deprecation
+cli: Deprecate the `-merge-policies` and `-merge-roles` flags from the `consul token update` command in favour of: `-add-policy-id`, `-add-policy-name`, `-add-role-name`, and `-add-role-id`.
```

Comment:
general question: Has there been discussion around "add" vs "append"? In the spike doc @cizh and I agreed that "append" was preferred since it implied appending to an existing list

Reply:
good catch, yeah append was agreed upon. this is my mistake